web/src/components/forms/update-project-form.tsx

```diff
@@ -47,6 +49,9 @@ export function UpdateProjectForm({
       }
     );
 
+    // call the callback to invalidate the project query
+    callback();
```

Comment:
This component doesn't know what this callback does - we shouldn't put a comment here to this effect as may confuse future us if a diff callback is used later which no longer does this.